wagtail/contrib/forms/tests/test_views.py

```diff
@@ -663,11 +663,11 @@ def test_list_submissions_csv_export(self):
         self.assertEqual(response.status_code, 200)
         data_lines = response.getvalue().decode().split("\n")
 
-        self.assertEqual(data_lines[0], 'Username,Submission date,Your email,Your message,Your choices\r')
+        self.assertEqual(data_lines[0], 'User email,Submission date,Your email,Your message,Your choices\r')
```

Comment:
Should this remain the username for regular users? If not, this would be a breaking change, wouldn't it?

Reply:
I changed this in the definition of `CustomFormPageSubmission` - see https://github.com/wagtail/wagtail/commit/87bc95f34e60c112f3b2c7052c66a06a57681c0f#diff-d2d058dc2ff8fff791e5b62fdd165cbd . I figured that since the intention of this test was to make sure we could capture data from `request.user` in the submission, it didn't really matter _which_ field of the user model we captured - I could have kept it closer to the original by doing

        form_data.update({
            'username': self.user.get_username(),
        })

but then we would have needed separate cases for the subsequent assertEqual lines that check the CSV data, depending on whether CustomUser or EmailUser was active. Changing it to consistently use the email field meant that we could avoid that.